src/containers/Tenant/Query/i18n/en.json

```diff
@@ -54,8 +54,8 @@
 
   "filter.text.placeholder": "Search by query text...",
 
-  "gear.tooltip": "Query execution settings have been changed for ",
-  "banner.query-settings.message": "Query results are displayed for ",
+  "gear.tooltip": "Query execution settings: ",
+  "banner.query-settings.message": "Query execution settings: ",
```

Comment:
```suggestion
  "gear.tooltip": "Query execution settings have been changed for ",
  "banner.query-settings.message": "Query was executed with modified settings: ",
```
Gear tooltip have only modified settings, your version doesn't mean that, I'd leave it as before.

As for the the banner, it relates to specific query execution, I think banner should mention it.

<img width="877" alt="Screenshot 2025-01-24 at 16 48 55" src="https://github.com/user-attachments/assets/f5306190-bcba-42ad-86b1-2a6d3d5e3b88" />